frontend/client/modules/proposals/actions.ts

```diff
@@ -31,6 +31,15 @@ function addProposalUserRoles(p: Proposal, state: AppState) {
   return p;
 }
 
+function addProposalVersionBool(p: Proposal) {
+  if (p.version === '2') {
+    p.isVersionTwo = true;
```

Comment:
It would be better to have this sent from the serializer instead of deriving it on the FE. 

```
...
    is_version_two = ma.Method("get_content")

    def is_version_two(self, obj):
        return True if version == '2' else False
```

Or, we could even just simply have a boolean "is_version_two" on the model itself.